img/h/hidrive.com.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 178.59 178.4" fill="#f80"><path d="M69.448 158.672c10.96 0 19.842 8.887 19.842 19.844H19.84C8.882 178.516 0 169.633 0 158.672v-69.45c10.96 0 19.843 8.884 19.843 19.842l-.002 39.687c0 5.479 4.443 9.921 9.922 9.921h39.685"/><path d="M39.684 128.907c0 5.48 4.443 9.922 9.921 9.922h79.37c5.48 0 9.92-4.443 9.92-9.922V49.535c0-5.48-4.44-9.922-9.92-9.922h-79.37c-5.478 0-9.92 4.443-9.92 9.922v79.372"/><path d="M109.131 19.77c-10.96 0-19.842-8.887-19.842-19.844h69.45c10.958 0 19.84 8.883 19.84 19.844v69.45c-10.96 0-19.843-8.884-19.843-19.842l.002-39.687c0-5.479-4.443-9.921-9.922-9.921h-39.685"/></svg>
```

Comment:
I don't see any site using the logo you provided. Isn't this a better one?
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 37 37" fill="#f80"><path d="M19 5c-3 0-6-2-6-5h19c3 0 5 2 5 5v19c-3 0-5-3-5-6V8c0-2-1-3-3-3"/><rect width="26.5" height="26.5" y="11" ry="3.1"/></svg>
```

Reply:
The logo I provided is the one specific to HiDrive. 
The image on the HiDrive website and the apps are different to the main Strato logo. I thought it would be better to use the HiDrive specific one rather than the general Strato logo to help differentiate between the products.
https://play.google.com/store/apps/details?id=com.strato.hidrive&hl=en
https://my.hidrive.com/#login

Reply:
So like this?
![](https://camo.githubusercontent.com/d22930627fc1e95b4b0290db587d6cca039ba6fb81ddef5a735ad97d6d4cb044/68747470733a2f2f642e74656d7066696c65732e646f776e6c6f61642f44363044304234323841393641392f3f703d6a5058335330364f6f6844)
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" fill="#f80" viewBox="0 0 48 48"><path d="M30 5c-3 0-6-2-6-5h19c3 0 5 2 5 5v19c-3 0-5-3-5-6V8c0-2-1-3-3-3"/><rect width="26" height="26" x="11" y="11" ry="3"/><path d="M18 43c3 0 6 2 6 5H5c-3 0-5-2-5-5V24c3 0 5 3 5 6v10c0 2 1 3 3 3"/></svg>
```